changelogs/fragments/6270-gconftool2-changed.yml

```
@@ -0,0 +1,5 @@
+bugfixes:
+  - gconftool2 - fix ``changed`` result always being ``true`` (https://github.com/ansible-collections/community.general/issues/6028).
+  - gconftool2 - remove requirement of parameter ``value`` when ``state=absent`` (https://github.com/ansible-collections/community.general/issues/6028).
+breaking_changes:
+  - gconftool2 - fix processing of ``gconftool-2`` when ``key`` does not exist, returning ``null`` instead of empty string (https://github.com/ansible-collections/community.general/issues/6028).
```

Maybe just something like:
```suggestion
  - gconftool2 - fix processing of ``gconftool-2`` when ``key`` does not exist, returning ``null`` instead of empty string. This return value ``previous_value`` was not documented previously (https://github.com/ansible-collections/community.general/issues/6028).
```

---

Actually `value` is also affected by it. I have just pushed an update.